ios/brave-ios/Sources/BraveVPN/Components/Region/BraveVPNRegionListView.swift

```diff
@@ -0,0 +1,260 @@
+// Copyright 2024 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import BraveShared
+import BraveStrings
+import GuardianConnect
+import SwiftUI
+
+public struct BraveVPNRegionListView: View {
+
+  @State
+  private var isAutomatic: Bool
+
+  @State
+  private var isLoading = false
+
+  @State
+  private var isRegionDetailsPresented = false
+
+  @State
+  private var isConfirmationPresented = false
+
+  @State
+  private var isShowingChangeRegionAlert = false
+
+  @State
+  private var selectedIndex = 0
+
+  @State
+  private var selectedRegion: GRDRegion?
+
+  @State
+  private var selectedRegionCities: [GRDRegion] = []
+
+  @State
+  private var regionModificationTimer: Timer?
+
+  public init() {
+    self.isAutomatic = BraveVPN.isAutomaticRegion
```

Comment:
This is invalid, `isAutomatic` is a `@State` var.  Assign it in `onAppear` instead

Reply:
Fixed at https://github.com/brave/brave-core/pull/24198/commits/db1d988f0db57643c18b4dd3cbca47d5c3adc98f